samples/I2C.js

```diff
@@ -121,27 +120,29 @@ function changeRGB(red, green, blue) {
 }
 
 function writeWord() {
-    resetCursor();
-
+    clear();
```

Comment:
This is the only way I currently can clear extra text that could appear as a result of I2C write failing.  In the future I'll need to create a way to either send the error to JS or make the I2C code smarter and have it try again if the write fails.